cli/cmd/runner.go

```diff
@@ -181,5 +181,8 @@ type runnerArgs struct {
 
 	lsClusterHideTerminated bool
 
+	// Application Version String
+	appVersion string
```

Comment:
convention on this project is to prefix these vars with the command that uses them